packages/store-ui/src/atoms/Select/Select.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import Select from '.'
+
+const optionsArray = [
+  ['great', 'Great'],
+  ['ok', 'Ok'],
+  ['bad', 'Bad'],
+]
+
+const mapPairToOption = (value: string, label: string) => {
+  return (
+    <option key={value} value={value}>
+      {label}
+    </option>
+  )
+}
+
+describe('Select', () => {
```

Comment:
Could add a test checking if the `data-store-select` exists?

Reply:
Isn't the first test it?

Reply:
right